common/src/main/resources/wynntils.accessWidener

```diff
@@ -1,44 +1,44 @@
-accessWidener v2 named
 
```

Comment:
why newline?

Reply:
my bad, i did not check properly after sorting.